app/scripts/modules/core/src/presentation/HoverablePopover.tsx

```diff
@@ -28,7 +28,10 @@ export interface IHoverablePopoverProps extends React.HTMLProps<any> {
   hOffsetPercent?: string;
   /** class to put on the popover content */
   className?: string;
-
+  /** class to put on the outermost element */
+  wrapperClassName?: string;
+  /** custom style attributes */
+  style?: React.CSSProperties;
```

Comment:
I feel some level of discomfort in adding more stuff to this component's API. That said, I ran into a bit of a wall because:
1) there is a mandatory wrapper div that doesn't get any classes right now, and in the flex layouts I'm working with adding layout classes to the outermost div is required to get things to fill space correctly
2) the very same wrapper div always gets `display: inline` slapped directly on it, which again is a no-go for the layouts I'm in which are flex all the way down and get totally screwed up by this

If people have other ideas, I'm happy to try to get these extra options out of here.

Reply:
Could the `style` prop be represented by another wrapper classname? Then only one prop is added.

Reply:
Sadly since the `display: inline` property gets added as an inline style prop inside `HoverablePopover` we'd have to use `!important` on the the property in the class. I'm not entirely opposed to the idea, but I'm not sure which is better — an `!important` sprinkled in a stylesheet that's invisible until you check the `.less` file, or an extra prop on this component's API that's explicit when you're reading the code 🤔

Reply:
You can move the `display: inline` from inline CSS to `HoverablePopover.css`  then add an override for MD. Something like 
```css
.HoverablePopover {
  display: inline;
}

.HoverablePopover.ManagedResourceStatusIndicator {
  display: flex;
}
```

Reply:
yeah, you're right it's worth trying this and seeing what happens — I was very anxious / risk-averse about unintended consequences from reducing the specificity, but maybe it's worth the risk to get that inline styling out of there

Reply:
https://github.com/spinnaker/deck/pull/7597